apm-application-toolkit/apm-toolkit-opentracing/src/main/java/org/apache/skywalking/apm/toolkit/opentracing/SkywalkingSpanBuilder.java

```diff
@@ -84,8 +83,10 @@ public SkywalkingSpanBuilder(String operationName) {
                 isExit = false;
             }
         } else if (Tags.PEER_HOST_IPV4.getKey().equals(key) || Tags.PEER_HOST_IPV6.getKey()
-                                                                                  .equals(key) || Tags.PEER_HOSTNAME.getKey()
-                                                                                                                    .equals(key)) {
+                                                                                  .equals(
+                                                                                      key) || Tags.PEER_HOSTNAME.getKey()
+                                                                                                                .equals(
+                                                                                                                    key)) {
```

Comment:
This format seems very strange, please fix.

Reply:
Done